### PR TITLE
Add Virtual-Assistant.yml

### PR DESCRIPTION
**Pull request by ritwik12**

### CODE REVIEW

**Comment by ritwik12** [comment]
@shiftkey and @dahlbyk  please review.It's been a long time

File: _data/projects/Virtual-Assistant.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Virtual Assistant
+desc: A linux based Virtual assistant on Artificial Intelligence in C  
+site: https://github.com/ritwik12/Virtual-Assistant
+tags:
+- C
+- Linux
+- Artificial Intelligence 
+upforgrabs:
+  name: jump in
+  link: https://github.com/ritwik12/Virtual-Assistant/labels/jump%20in
```
Comment (shiftkey) [inline _data/projects/Virtual-Assistant.yml:10]: There aren't any issues using this label - is that intentional?

Comment (ritwik12) [reply _data/projects/Virtual-Assistant.yml]: yes @shiftkey

Comment (ritwik12) [reply _data/projects/Virtual-Assistant.yml:10]: Missed it.Done now, Thanks

**Pull request merged**